apps/common/src/main/scala/org/lfdecentralizedtrust/splice/store/StoreErrors.scala

```diff
@@ -14,6 +15,10 @@ trait StoreErrors {
     Status.NOT_FOUND.withDescription("No matching log indices found").asRuntimeException
   }
 
+  def contractIdNotFound(cid: PrettyContractId) = {
+    Status.NOT_FOUND.withDescription(s"Contract id not found: $cid").asRuntimeException
```

Comment:
```suggestion
    Status.NOT_FOUND.withDescription(show"Contract id not found: $cid").asRuntimeException
```
I don't think this looks nice otherwise